### DISCUSSION TITLE
公共模板更新失败错误代码1006

### DISCUSSION THREAD

**Discussion by Suntxx** [qd-today/templates#641] [Q&A] [answered]
新手装好后公共模板1006错误代码，请问该怎么解决呢

![image](https://github.com/qd-today/templates/assets/113056239/08d55e18-2fb1-4ed2-8577-34a6ddf7eb11)

**Comment by blgs11** [comment, marked as answer]
https://github.com/qd-today/qd/issues/467 read this
